views/calendar/card.hbs

```diff
@@ -43,7 +43,7 @@
               <span class="muted video-conference" style="display: none;">{{$t "calendar.text.videoConferenceDetailsAreLoading" }}</span>
               <!-- state: RUNNING -->
               <i class="fa fa-video-camera video-conference join-conference" aria-hidden="true" style="display: none;">&nbsp;</i>
-              <a class="video-conference join-conference" href="/videoconference" style="display: none;">{{$t "calendar.text.participateInVideoConference" }}</a>
+              <a class="video-conference join-conference" href="/videoconference" style="display: none;" data-testid="participate-video-conf-link">{{$t "calendar.text.participateInVideoConference" }}</a>
```

Comment:
Just to be consistrnt, please use the full "conference" instead of only "conf"

Reply:
yes, updated it to 'conference'.